day_7/fizzbuzz.js

```diff
@@ -0,0 +1,24 @@
+/* make a loop for 1-100 where the reminder if 0 will log fizzbuz,
+buzz or fiz, */
+fizzBuzznumber(-100);
+var numberPlus = (var i = 0; i <= number; i++)
```

Comment:
On line 4 and 5 you have duplicate `numberPlus` variables declared. You can only declare the variable once. _You can however, reassign it afterwards._